pg_search/tests/pg_regress/expected/mixedff_advanced_06_score_function.out

```diff
@@ -317,10 +317,16 @@ CREATE TABLE score_test (
     is_featured BOOLEAN
 );
 -- Insert test data with deterministic values
+--
+-- each new row contains a (predictable) set of random words.
+-- This is intentional to produce different scores for each matching
+-- result of the queries that follow so that the test itself doesn't
+-- have to worry about tie-breaking on same-scores.
+--
```

Comment:
Basically, what the comment says.  It has an effect on the `score` column of all the queries in the test, but they'll now be the same from run-to-run and not be affected by our lack of tie-breaks in TopN.

/cc @stuhood / @mdashti